src/css/markdown.scss

```diff
@@ -4,15 +4,15 @@ div.markdown {
 
   ul {
     padding-left: var(--ifm-list-left-padding);
-    li {
```

Comment:
For this, we'll still want the disc when a it's first child and circle when indented further. 

This way not all bullets become circles like we have with this change as is:
https://share.cleanshot.com/3Mk98nzB

Reply:
Oh yeah, missed that.

Reply:
fixed here [d2ce38e](https://github.com/cypress-io/cypress-documentation/pull/5572/commits/d2ce38e5a3a7a59e7a1d5c974d1b248b1bd3f78d)